### ISSUE TITLE
No logs: "ERROR: Slack notification failed. See Jenkins logs for details."

### ISSUE THREAD

**Issue by KristianWindsor** [jenkinsci/slack-plugin#534]
When I click `Test Connection`, the logs successfully show up in Jenkins.

But when I try to send a message using scripted pipeline, nothing at all is logged to Jenkins. 
This is all that is logged to the job's console output:
```
run slackstepsend, step false, desc false
Slack Send Pipeline step configured values from global config - baseUrl: true, teamDomain: true, token: false, channel: false, color: false
ERROR: Slack notification failed. See Jenkins logs for details.
```

What can I do? I have no idea if it's syntax, or the channel can't be found, or if the token is invalid. I'm debugging blind!

**Comment by timja** [comment]
Check the jenkins system logs, if there's not enough info there then add a log recorder for this class at FINE level:

https://github.com/jenkinsci/slack-plugin/blob/master/src/main/java/jenkins/plugins/slack/StandardSlackService.java

**Comment by timja** [comment]
Should be fixed or at least better logging in 2.19
https://github.com/jenkinsci/slack-plugin/pull/541
